plenum/test/pp_seq_no_restoration/test_backup_primary_restores_pp_seq_no_if_view_is_same.py

```diff
@@ -28,10 +30,10 @@ def test_backup_primary_restores_pp_seq_no_if_view_is_same(
     # Send some 3PC-batches and wait until the replica orders the 3PC-batches
     sdk_send_batches_of_random(looper, txnPoolNodeSet,
                                sdk_pool_handle, sdk_wallet_client,
-                               num_reqs=7, num_batches=7,
+                               num_reqs=7, num_batches=num_batches,
                                timeout=tconf.Max3PCBatchWait)
 
-    seq_no = 7 if view_no == 0 else 8
+    seq_no = num_batches if view_no == 0 else 2 * num_batches + 1 + tconf.Max3PCBatchWait
```

Comment:
Is this formula correct? `Max3PCBatchWait` is seconds between batches, why do we use to calculate the seq_no?